docs/pages/development/introduction.md

```diff
@@ -4,46 +4,26 @@ title: Introduction
 
 import ImageSpotlight from '~/components/plugins/ImageSpotlight'
 
-> ⏩ Want to create a development build for your project? Follow the [Getting Started guide](getting-started.md).
+> ⏩ Ready to advance from Expo Go and get a development build of your project? Follow the [Getting Started guide](getting-started.md).
 
-> 👀 Want to get notified of new releases with a changelog and upgrade instructions? Sign up for the [mailing list](https://expo.dev/mailing-list/dev-client).
+> 👀 Want to get notified of new releases to `expo-dev-client` with a changelog and upgrade instructions? Sign up for the [mailing list](https://expo.dev/mailing-list/dev-client).
 
-Building your project with Expo allows you to iterate on your app quickly and safely by allowing you to make most changes in JavaScript. [This workflow](https://blog.expo.dev/javascript-driven-development-with-custom-runtimes-eda87d574c9d) allows your team to:
+Building your project with Expo allows you to iterate quickly and safely by allowing you to make most changes in JavaScript. [Your team can acheive web-like iteration speeds](https://blog.expo.dev/javascript-driven-development-with-custom-runtimes-eda87d574c9d) by decomposing your application into:
 
-- Easily onboard new developers without native experience
-- Quickly verify the impact of changes to your project
-- Develop iOS apps without needing a machine running MacOS
+- **A Client**: A native binary that can interact with platform API which is built and distributed the same as in traditional native development. 
+- **An Update**: An atomic collection of assets like JavaScript, data files, or media that instruct the client how to behave. These updates may be served from your local computer by `expo-cli`, embedded in the binary by EAS Build, or hosted on a publicly available server.
 
-When first starting your project, the standard runtime provided in Expo Go was likely sufficient. As your project moves toward release, you may find that you need to customize your project, either to reduce your bundle size, to use a module offered by developers in the React Native community, or even to add your own custom native code. At that point, you can create a development build of your app, install it on your phone, and continue developing.
+## Development Clients: from Expo Go to Development Builds
```

Comment:
```suggestion
## From Expo Go to Development Builds
```
I think having both the terms "Development Clients" and "Development Builds" in the same header is a little confusing (would be reasonable to ask "what's the difference?")